services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/controller/VpcController.java

```diff
@@ -25,10 +25,16 @@
 import com.futurewei.alcor.vpcmanager.entity.VpcState;
 import com.futurewei.alcor.vpcmanager.entity.VpcStateJson;
 import com.futurewei.alcor.vpcmanager.utils.RestPreconditionsUtil;
+import com.futurewei.alcor.vpcmanager.entity.RouteWebJson;
+import com.futurewei.alcor.vpcmanager.entity.RouteWebObject;
```

Comment:
would the code look better if we just do:
import com.futurewei.alcor.vpcmanager.entity.*;

Reply:
See more than 2 references in the package, it will be switched to * as the current code optimization pattern is.